Dockerfile

```
@@ -1,28 +1,11 @@
-FROM alpine/git:v2.30.2 AS getter
 
-RUN mkdir /pygeofilter
-WORKDIR /pygeofilter
-# switch to geopython/pygeofilter once bug fixes are merged via PR
-RUN git clone --single-branch --branch main https://github.com/sparkgeo/pygeofilter.git
-WORKDIR /pygeofilter/pygeofilter
-RUN git checkout 5cfa8afbb6ffc8a30b8dde5be7a2981d4b534678
+FROM python:3.8-slim
 
+RUN apt-get update && apt-get -y install python3-pip python3-psycopg2 libpq-dev
+RUN rm -rf /var/lib/apt/lists/*
```

Docker uses a diff based file system. A "layer" gets created after each command in a Dockerfile.
This `rm` will remove the files listed, but the files were written to the previous layer. So the final image size will be the same (technically adding the `rm` makes it a very small amount larger).
To get the desired effect here, the two `RUN` statements need to be combined.

```suggestion
RUN apt-get update && apt-get install -y \
        libpq-dev \
        python3-pip \
        python3-psycopg2 \
    && rm -rf /var/lib/apt/lists/*
```